apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/SizedBufferingConsumer.cs

```diff
@@ -35,13 +34,13 @@ internal class SizedBufferingConsumer<T> : IFlushableConsumer<T>
         /// <summary>The size of the buffer in bytes.</summary>
         private readonly int _bufferSize;
 
-        /// <summary>The buffered items.</summary>
-        private IList<T> _items;
+        /// <summary>The buffered items.</summary>		
```

Comment:
Nit: remove trailing whitespace.
Possibly document why this isn't read-only? (So that we can replace it with the next buffer when we need to, I realize.)